src/System.Net.Sockets/tests/FunctionalTests/Accept.cs

```diff
@@ -289,6 +289,44 @@ public async Task AcceptAsync_MultipleAcceptsThenDispose_AcceptsThrowAfterDispos
                 }
             }
         }
+
+        [Fact]
+        public async Task SyncAcceptGetsCancelledByDispose()
+        {
+            if (!UsesSync)
```

Comment:
Shouldn't the test also succeed for async?  That's usually the easier case.

Reply:
Yes, it should.